app/components/telegram-auth.tsx

```diff
@@ -391,6 +396,11 @@ export default function TelegramAuth() {
     )
   }
 
+  async function resetLoginState() {
+    await logout()
```

Comment:
Does just logging out of Telegram solve it?
We can test to see if that’s enough, but if you want to take a look, check out `handleLogOutClick` in `menu.tsx`, it handles clearing all the state.

Reply:
yea - I'm hoping it does - I am curious, in `handleLogOutClick` it seems like this is the only update to localStorage - is that right? aren't we storing more things in there? or are those all cleared out by the telegram logout? I don't want to just clear EVERYTHING out of local storage so that, eg, other users on android don't also get logged out, but I suspect there might be some state this logout function is not clearing that should be cleared...

Reply:
ok cool - I'm hopeful that just clearing out the telegram auth state gets the user past this and other error states - I'd say if we continue to see these issues we should try harder to clear state here - hoping to get a little more data and buyin from the rest of the team on the right strategy soon so I'll leave it as-is for now, but I've made a TODO for myself to come back to it